### PR TITLE
Added install-base command for pre-existing projects

### PR DESCRIPTION
**Pull request by MaxLeiter**

### CODE REVIEW

**Comment by SirCmpwn** [comment]
This should also add symlinks for `/bin/launcher` and `/bin/threadlist` (and probably `/bin/explorer`).

**Comment by MaxLeiter** [comment]
symlinks from `/.knightos/pkgroot/bin/x` to where exactly? For what purpose?

**Comment by SirCmpwn** [comment]
See the KnightOS makefile.

File: project.py [hunk 1]
```diff
@@ -153,3 +159,12 @@ def findroot():
             path = os.path.realpath(os.path.join(path, ".."))
     stderr.write("There doesn't seem to be a KnightOS project here. Did you run `knightos init`?\n")
     exit(1)
+
+#Currently there's no way to overwrite a pre-existing symlink
+def force_symlink(file1, file2):
+    try:
+        os.symlink(file1, file2)
+    except OSError as e:
```
Comment (Ivoah) [inline project.py]: Couldn't you just start off by deleting file2? Will it throw an error if you try to delete a non-existant file?

**Pull request merged**